src/platform/intel/ace/platform.c

```diff
@@ -21,6 +21,10 @@
 #include <ipc/info.h>
 #include <kernel/abi.h>
 #include <rtos/clk.h>
+#include <adsp_memory.h>
```

Comment:
typos in the commit message after the sign-off

Reply:
Are there? Can you please point me where? Checkpatch finds no typos...